subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MultiProjectJvmApplicationInitIntegrationTest.groovy

```diff
@@ -52,13 +54,20 @@ abstract class AbstractMultiProjectJvmApplicationInitIntegrationTest extends Abs
         targetDir.file(settingsFile).exists()
         !targetDir.file(buildFile).exists()
 
-        targetDir.file(incubating ? "build-logic" : "buildSrc").assertHasDescendants(
+
+        def buildLogicFolder = targetDir.file(incubating ? "build-logic" : "buildSrc")
+
+        def commonConventionsPath = "src/main/${dsl.id}/some.thing.${dsl.fileNameFor("${language}-common-conventions")}"
+        buildLogicFolder.assertHasDescendants(
             settingsFile,
             buildFile,
-            "src/main/${dsl.id}/some.thing.${dsl.fileNameFor("${language}-common-conventions")}",
+            commonConventionsPath,
             "src/main/${dsl.id}/some.thing.${dsl.fileNameFor("${language}-application-conventions")}",
             "src/main/${dsl.id}/some.thing.${dsl.fileNameFor("${language}-library-conventions")}",
         )
+        buildLogicFolder.file(commonConventionsPath).assertContents(
+            containsText("JavaLanguageVersion.of(" + Jvm.current().javaVersion.majorVersion + ")")
```

Comment:
Can we perform a stronger assertion here? IMO we should be testing for the existence of the entire `toolchain` block and its contents as opposed to testing that `JavaLanguageVersion` exists _somewhere_ in the file without the additional context

Reply:
Ahh I see. I'm fine the way it is then